src/main/scala/viper/silver/plugin/standard/inline/InlineRewrite.scala

```diff
@@ -100,18 +101,17 @@ trait InlineRewrite {
     * Removes given predicate unfolds and folds from statement.
     *
     * @param stmts A Seqn whose statements will be traversed
-    * @param unfoldPreds A set of the string names of the precondition predicates to not unfold
-    * @param foldPreds A set of the string names of the postcondition predicates to not fold
+    * @param predIds The set of predicates for which we will remove unfold and fold statements
```

Comment:
I guess it _kinda_ makes sense to remove any `folds` for which the `unfold` was removed, otherwise there wouldn't be anything to fold back...if I'm understanding this correctly.